docs/management/index-lifecycle-policies/create-policy.asciidoc

```diff
@@ -45,7 +47,10 @@ smaller {ref}/indices-segments.html[segments].
 * *Cold.* The index is no longer being updated and is seldom queried, but is 
 still searchable. If you have a big deployment, you can move it to even 
 less performant hardware. You might also reduce the number of replicas because 
-you expect the data to be queried less frequently.
+you expect the data to be queried less frequently. To keep the index searchable 
+for a longer period, and reduce the hardware requirements, you can use the 
+{ref}/frozen-indices.html[freeze action]. Queries are slower on a frozen index because the index is 
+reloaded from the drive to RAM on demand.
```

Comment:
`the drive` sounds weird to me.  Mayne just `disk`?

Reply:
++ for disk